benchmarks/TFLite/CMakeLists.txt

```diff
@@ -194,6 +196,8 @@ iree_benchmark_suite(
     "dylib"
   RUNTIME_FLAGS
     "--task_topology_group_count=1"
+
+  COLLECT_COMPILATION_STATISTICS
```

Comment:
This is going to be a lot of statistics. Is each statistic for each benchmark configuration a new series?

The PR comment shows "[Top 3 out of 332 statistic results showed]". Isn't that going to be really difficult to sift through on the dashboard?

Can we group the statistics or prune the set of configurations that track them?

(Writing to a database then querying that database will offer much more flexibility than continuing to push adhoc updates to https://github.com/google/dana IMO)